### PR TITLE
feat(react): switch to vite and @azure/msal

### LINKED ISSUE #124: Use `@azure/msal-react`
Switch from `@intility/react-msal-browser` to `@azure/msal-react` in Create React App templates.

### PR DESCRIPTION
**Pull request by h3rmanj**
- create-react-app -> vite
- jest -> vitest
- @intility/react-auth -> @azure/msal-react
- better structure
- better comments on template code
- remove javascript template

Try the new template by running (creates from branch version)

```
npx degit intility/templates/react#react-vite intility-vite-app
cd intility-vite-app

npm install
npm start
```

run the docs by cloning and running
```
cd docusaurus
npm install
npm start
```

closes #124

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,16 +25,14 @@ More frameworks might be included in the future.
 
 To get started with a project, simply run one of the following commands.
 
-CRA TypeScript:
+Vite + React + TypeScript:
```
Comment (JonasKs) [inline README.md:28]: Add `Bifrost`?

File: docusaurus/express/gettingStarted.mdx [hunk 2]
```diff
@@ -17,16 +17,15 @@ yo @intility/express                        # Use Yeoman CLI to run the template
...
 2. Do you want to include **demo endpoints** in the generated project?
-    - This is recommended for first time setup or if you are not familiar with Express API architecture 
-    - These files provides example CRUD operations with a router-, controller- and service-layer
-3. **Automatic Git initialization** and push to given Git repository.
-    - Please [create a GitLab](https://gitlab.intility.com/projects/new?namespace_id=978#blank_project) project, and get the SSH key. 
-    - NOTE: Unselect the: 'Initialize repository with a README' under 'Project Configuration'
-:::
+   - This is recommended for first time setup or if you are not familiar with Express API architecture
+   - These files provides example CRUD operations with a router-, controller- and service-layer
+3. **Automatic Git initialization** and push to given Git repository. - Please [create a GitLab](https://gitlab.intility.com/projects/new?namespace_id=978#blank_project) project, and get the SSH key. - NOTE: Unselect the: 'Initialize repository with a README' under 'Project Configuration'
```
Comment (JonasKs) [inline docusaurus/express/gettingStarted.mdx:27]: Dead URL? `https://gitlab.intility.com/projects/new` works, but not with `?namespace_id=978#blank_project`

**Review by JonasKs — APPROVED** [review]

**Comment by PerAndersS** [comment]
Awesome!

One small thing i noticed is when running the `npm run build` I got the following message. Not sure what this means tho :)
image

**Review by PerAndersS — APPROVED** [review]

**Comment by h3rmanj** [comment]
> Awesome!
> 
> One small thing i noticed is when running the `npm run build` I got the following message. Not sure what this means tho :) image

I ran `source-map-explorer` to check it out.
image

I'll have to look into bifrost and its dependencies, as it seems it can't shake unused components.

Most of it comes from `@azure/msal`. I have seen issues there mentioning their size, but it would seem a lot of it is needed, and potential improvements won't come until the next major.

Many dependencies could also be code-splitted, depending on each applications needs, but I fear it would be too cluttered with fallbacks etc in the template alone.

250kb isn't optimal, but it's not the worst for internal apps which usually always is behind an authwall.

**Pull request merged**